src/embed/connection.rs

```diff
@@ -272,6 +274,18 @@ async fn do_put(txn: &RwLock<Transaction<'_>>, req: PutRequest) -> Result<PutRes
     Ok(PutResponse {})
 }
 
+// This fn just forwards to sync::begin_sync at this point. execute() requires its
+// do_ers to take txns even if they don't need it so this fn also helps begin_sync
+// have a simpler signature.
+async fn do_begin_sync<'a, 'b>(
+    _store: &'a dag::Store,
```

Comment:
[style] Just name these params `_`. I believe that is idiomatic in Rust to say that something is not used, and in that case don't even need the comment.